searchguard/rolesmapping.py

```diff
@@ -0,0 +1,157 @@
+#!/usr/bin/python3
+
+import requests
+import json
+import searchguard.settings as settings
+from searchguard.exceptions import *
+from searchguard.roles import check_role_exists
+
+
+def __process_data(role, properties):
```

Comment:
Nit picky, but shouldn't this method have only one leading underscore?  [PEP](https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles)

Reply:
I think both are allowed, but one underscores = should not call it, two = really should not call it.

Reply:
[PEP](https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles):
-`_single_leading_underscore`: weak "internal use" indicator.
-`__double_leading_underscore`: when naming a class attribute, invokes name mangling.